apps/centralcms/.gitignore

```diff
@@ -0,0 +1,43 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
I think we don't need this file but let's just update the root directory

Reply:
I thought I removed this, it was auto-generated when I created the project instance 🤦